builder/ecs/image_config.go

```diff
@@ -151,7 +151,7 @@ type AlicloudImageConfig struct {
 	AlicloudImageIgnoreDataDisks bool `mapstructure:"image_ignore_data_disks" required:"false"`
 	// The region validation can be skipped if this value is true, the default
 	// value is false.
-	AlicloudImageSkipRegionValidation bool `mapstructure:"skip_region_validation" required:"false"`
+	AlicloudImageSkipRegionValidation bool `mapstructure:"image_skip_region_validation" required:"false"`
```

Comment:
How does this differ from skip region validation being set in the access config https://github.com/hashicorp/packer-plugin-alicloud/blob/480ef331f1a2c4c2bf6762ac2882459da1c8c09f/builder/ecs/access_config.go#L39?

Should this single attribute apply to all region validation settings?

Reply:
This is a good point. Only one of the attributes is being used in the code as it stands, so we can remove the one that is only defined and never used. This will take care of the conflict, and not break anything.